treeherder/perf/utils.py

```diff
@@ -0,0 +1,31 @@
+import datetime
+
+MON, TUE, WED, THU, FRI, SAT, SUN = range(1, 8)
+
+
+def calculate_time_to_triage(created, due_days=3):
```

Comment:
due_days is never used in calls to this or tests.  Not sure if this is needed, but it would make the tests more robust.

Reply:
i'll look into using it in tests. thanks